RecoLocalCalo/HGCalRecAlgos/src/ClusterTools.cc

```diff
@@ -83,14 +85,16 @@ math::XYZPoint ClusterTools::getMultiClusterPosition(const reco::HGCalMultiClust
   for( const auto& ptr : clu.clusters() ) {   
     const double x = ptr->x();    
     const double y = ptr->y();
-    const float point_r = std::sqrt(x*x + y*y);    
+    const double point_r2 = (x*x + y*y);    
```

Comment:
the reason for x,y,z and energy to be in double is debatable on the storage side.
there is nothing else inside this loop that requires double.

OK for now